runtime/autoload/health/provider.vim

```diff
@@ -502,7 +502,14 @@ function! s:check_node() abort
           \ ['Install Node.js and verify that `node` and `npm` commands work.'])
     return
   endif
-  call health#report_info('Node: '. s:system('node -v'))
+  let node_v = get(split(s:system('node -v'), "\n"), 0, '')
+  call health#report_info('Node: '. node_v)
+  if !s:shell_error && s:version_cmp(node_v[1:], '6.0.0') < 0
+    call health#report_warn('nodejs host does not support '.node_v)
+  endif
+  if has('win32') || !provider#node#can_inspect()
+    call health#report_warn('nodejs on this system does not support --inspect-brk or this system is Windows so $NVIM_NODE_HOST_DEBUG is ignored.')
```

Comment:
The message is getting too long, there's not much reason to use the same message when we can trivially check if we're on Windows.

```vim
if has('win32')
  call health#report_warn('nodejs on Windows does not support --inspect-brk. $NVIM_NODE_HOST_DEBUG will be ignored.')
elseif !provider#node#can_inspect()
  call health#report_warn('nodejs on this system does not support --inspect-brk. $NVIM_NODE_HOST_DEBUG will be ignored.')
endif
```

Reply:
As long as the version requirement is reached, `--inspect-brk` works on Windows.
The problem is that you cannot pass this to `node` through the shims.

```sh
# Ignore the shims and run cli.js directly
node   $PREV  "$(npm root -g)/neovim/bin/cli.js"   $NEXT
```

Without calling `node` directly, the user can't set `$PREV`.